### PR TITLE
[DO-2106]Made convert_to_usd utility

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: data_ocean/utils.py [hunk 1]
```diff
@@ -155,7 +155,1052 @@ def replace_incorrect_symbols(string):
...
-        record_to_string += (element.tag or 'NONE') +': ' + (element.text or 'None') + '; '
+        record_to_string += (element.tag or 'NONE') + ': ' + (element.text or 'None') + '; '
     log = open(filename, 'a')
     log.write(record_to_string)
     log.close()
+
+
+#rates are on December, 31 from http://openexchange.org/
+def convert_to_usd(currency: str, amount: float, year: int):
+    rates_to_usd = {
```
Comment (criticus) [inline data_ocean/utils.py:166]: consider moving the dictionary to a separate file

**Comment by criticus** [comment]
besides moving the dictionary to a separate file it looks good

**Review by criticus — APPROVED** [review]

File: data_ocean/utils.py (hunk 1 above)
Comment (IegorSoboliev) [reply data_ocean/utils.py:166]: done

Comment (criticus) [reply data_ocean/utils.py]: great, more readable now 👍

**Pull request merged**